content/metadata.yaml

```
@@ -808,7 +808,7 @@ authors:
       confirmed: yes
     coi:
       string: "None"
-      lastapproved: !!str 2020-11-11
+      lastapproved: !!str 2020-08-02
```

```suggestion
      lastapproved: !!str 2021-08-02
```
I also missed the year the first time I looked at it...

---

Thanks @rando2 !